query/src/datasources/database/system/processes_table.rs

```diff
@@ -47,7 +47,7 @@ impl ProcessesTable {
             DataField::new("state", DataType::String, false),
             DataField::new("database", DataType::String, false),
             DataField::new("extra_info", DataType::String, true),
-            DataField::new("memory_usage", DataType::UInt64, true),
+            DataField::new("memory_usage", DataType::Int64, true),
```

Comment:
why int64?

Reply:
Object may be created by thread 1 and destroy by thread 2